### ISSUE TITLE
RDPwrap is blocked by Chrome

### ISSUE THREAD

**Issue by cdlvcdlv** [stascorp/rdpwrap#592]
Both .msi and .zip installs are blocked by Chrome.

It says is dangerous/malicious.

Edit: the same thing with Firefox. It says contains virus.

**Comment by binarymaster** [comment]
Duplicate of #277.
